modules/doc/content/getting_started/examples_and_tutorials/tutorial04_meshing/step10_rgmb.md

```diff
@@ -0,0 +1,143 @@
+
+### RGMB Cartesian Assembly Example
+
+- Tips:
+
+  - Elements belonging to a unique pin cell will be automatically provided a unique `pin_id`
+  - Ducts for Cartesian assemblies are not currently supported
+
+!media tutorial04_meshing/rgmb_assemblymesh_cart_blockid.png
+       id=tutorial04-rgmb_assemblymesh_cart_blockid
+       caption=Cartesian assembly colored by `subdomain_id` (left), `region_id` (middle), and `pin_id` (right). Subdomain ids categorize elements as belonging to different subdomains (blocks), region ids categorize elements belonging to different regions (material zones), and `pin_ids` categorize elements belonging to different pin cells.
+       style=width:75%;display:block;margin-left:auto;margin-right:auto;
+
+!listing sec10_rgmb_simple_examples/rgmb_core_cartesian.i
+         id=tutorial04-rgmb_rgmb_core_cartesian
+         caption=Example RGMB Cartesian assembly.
+         block=Mesh/assembly1
+
+### RGMB Hexagonal Assembly Example
+
+[AssemblyMeshGenerator.md] calls [PatternedHexMeshGenerator.md] or [PatternedMeshGenerator.md] to generate a Cartesian or hexagonal lattice of pin-like structures.
+
+- Tips:
+
+  - Hexagonal ducts and background coolant region can be added to hexagonal assemblies. The region IDs for the duct and background regions are set by the `duct_region_ids` and `background_region_id` parameters.
+  - Don't want explicitly defined pins in your assembly? See [step11_rgmb_homo.md] for how to define homogenized assemblies with [AssemblyMeshGenerator.md].
+
+!media tutorial04_meshing/rgmb_assemblymesh_hex.png
+       id=tutorial04-rgmb_assemblymesh_hex
+       caption=Hexagonal assembly colored by `subdomain_id` (left), `region_id` (middle), and `pin_id` (right). Subdomain ids categorize elements as belonging to different subdomains (blocks), region ids categorize elements belonging to different regions (material zones), and `pin_ids` categorize elements belonging to different pin cells.
+       style=width:75%;display:block;margin-left:auto;margin-right:auto;
+
+!listing sec10_rgmb_simple_examples/rgmb_core_hexagonal.i
+         id=tutorial04-rgmb_rgmb_core_hexagonal
+         caption=Example RGMB hexagonal assembly.
+         block=Mesh/assembly1
+
+## CoreMeshGenerator
+
+[CoreMeshGenerator.md] calls [PatternedHexMeshGenerator.md] or [PatternedMeshGenerator.md] to generate a Cartesian or hexagonal lattice of assembly-like structures.
+
+Defining `dummy_assembly_name` and using it in inputs defines a dummy assembly in the core
+
+### Cartesian Core Example
+
+!media tutorial04_meshing/rgmb_coremesh_cart.png
+       id=tutorial04-rgmb_coremesh_cart
+       caption=Cartesian core colored by `subdomain_id` (left), `region_id` (middle), and `assembly_id` (right).
+       style=width:75%;display:block;margin-left:auto;margin-right:auto;
+
+!listing sec10_rgmb_simple_examples/rgmb_core_cartesian.i
```

Comment:
@shikhar413 seems like the input "empty" and dummy_assembly_name= empty should be removed here as they do not do anything.